src/bodywork/git.py

```diff
@@ -25,12 +25,13 @@
 from subprocess import run, CalledProcessError
 
 from .constants import DEFAULT_PROJECT_DIR, SSH_DIR_NAME, SSH_GITHUB_KEY_ENV_VAR
+from .logs import bodywork_log_factory
 
 
 def download_project_code_from_repo(
-    url: str,
-    branch: str = 'master',
-    destination: Path = DEFAULT_PROJECT_DIR
+        url: str,
```

Comment:
Don't need the tabs - see 3rd code block [here](https://black.readthedocs.io/en/stable/the_black_code_style.html#code-style).

It's a bit odd, but you get used to it.

Reply:
I didn't touch this code, not sure why it's come up as changed. I have fixed it.

Reply:
Odd